### DISCUSSION TITLE
wagmi instead on onboard

### DISCUSSION THREAD

**Discussion by thelostone-mc** [gitcoinco/grants-stack#213] [Q&A] [answered]
## Description

- Depreacated wallet link which means we won't be able to use hardware wallet https://github.com/blocknative/web3-onboard/tree/v2-web3-onboard-develop/packages/walletlink
- Passport team had implemented this and did recommend that https://wagmi.sh/ is simpler and easier to integrate
- onboard has the styling shadow dom which means it is self contained and this prevents us from styling how the modal looks. 

The proposal here is to use [wagmi.sh](https://wagmi.sh/) instead which has been recommended by the passport team
It is bare backbones which means we have complete control over styling and we can choose what to add / remove
Additionally this code could be written once and then shared between passport /grant hub / round manager / round explorer (aka low effort once we do the intial setup)

**Comment by DanieleSalatti** [comment, marked as answer]
Just wanted to mention that wagmi.sh integrates nicely with [RainbowKit](https://www.rainbowkit.com/) too

**Reply by thelostone-mc** [reply]
Ended up doing this 🙌🏼  thanks @DanieleSalatti

**Comment by vacekj** [comment]
Closing as stale.
